org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/problem/messages.properties

```diff
@@ -906,7 +906,7 @@
 1106 = Preview features enabled at an invalid source release level {0}, preview can be enabled only at source level {1}
 1107 = The Java feature ''{0}'' is only available with source level {1} and above
 1108 = You are using an API that is part of a preview feature and may be removed in future
-1109 = Compiling for Java version ''{0}'' is no longer supported. Minimal supported version is ''{1}''
+1109 = Java language version ''{0}'' is not supported, using version ''{1}'' instead
```

Comment:
Those are two different problems that need different reporting:
A user trying to compile for Java 1.3 is in an entirely different situation than a user trying to compile for Java 30.

(we already had quite some discussion about this message when older versions were dropped).

Reply:
Actually, the new message text would be wrong for the original error, which doesn't use latest version instead but *aborts*.

Reply:
OK. It seems that we'd need to create a new dedicated problem id entirely then. Or is there anything better to try?

Reply:
I've created a different problem and message.